pypi/pip_generate.go

```diff
@@ -93,7 +93,7 @@ func (w *wheelInfo) makeBazelRule(name *string, wheelDir *string) string {
 		output += fmt.Sprintf("        licenses=[\"notice\"],\n")
```

Comment:
But now `wheelDir` is an absolute path, so the generated `filegroup` will be invalid, right...? This is why I think it needs to be specified relative to the repo root. (This comment actually applies a few lines up but GitHub sucks and won't let me put it there.)